cpu-on-dns-pods-high-variables.tf

```diff
@@ -0,0 +1,66 @@
+variable "cpu_on_dns_pods_high_enabled" {
+  type    = bool
+  default = true
+}
+
+variable "cpu_on_dns_pods_high_warning" {
+  type    = number
+  default = 70
+}
+
+variable "cpu_on_dns_pods_high_critical" {
+  type    = number
+  default = 85
+}
+
+variable "cpu_on_dns_pods_high_evaluation_period" {
+  type    = string
+  default = "last_5m"
```

Comment:
Did I leave this threshold for just 5m? I think this should be longer. 30 mins perhaps.